modules/os2forms_fasit/os2forms_fasit.services.yml

```diff
@@ -1,7 +1,8 @@
 services:
   Drupal\os2forms_fasit\Helper\Settings:
     arguments:
-      - "@keyvalue"
+      - "@config.factory"
+      - "@key.repository"
```

Comment:
https://www.drupal.org/docs/drupal-apis/services-and-dependency-injection/services-and-dependency-injection-in-drupal#s-autowiring-services

Reply:
Used autowiring in fa892f9746d714a6f73099684fb9e86d5180ef69.